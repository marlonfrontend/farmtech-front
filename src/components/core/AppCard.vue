<script setup lang="ts"></script>

<template>
  <div class="bg-white shadow border p-10 rounded text-center md:-mt-24">
    <h3 class="text-lg font-semibold font-serif mb-5">Filtrar dados sobre um pa√≠s</h3>
  </div>
</template>
