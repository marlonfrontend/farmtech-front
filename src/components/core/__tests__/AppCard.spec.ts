import { mount } from '@vue/test-utils'
import { describe, expect, it } from 'vitest'
import AppCard from '../AppCard.vue'

describe('AppCard.vue', () => {
  it('should render component props', () => {
    const props = {
      title: 'Card Title'
    }
    const wrapper = mount(AppCard, {
      propsData: { ...props }
    })
    expect(wrapper.props()).toContain(props)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
