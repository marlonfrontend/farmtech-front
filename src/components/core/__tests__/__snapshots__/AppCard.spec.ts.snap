// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`AppCard.vue > should render component props 1`] = `
"<div class=\\"bg-white shadow border p-10 rounded\\">
  <h3 class=\\"text-lg md:text-xl text-center font-semibold font-serif mb-5\\">Card Title</h3>
</div>"
`;
